.vscode/settings.json

```diff
@@ -74,7 +74,13 @@
     "psi-header.changes-tracking": {
         "autoHeader": "autoSave",
         "exclude": [
-            "css", "json", "xml", "config", "plaintext", "markdown", "batch"
+            "css",
+            "json",
+            "xml",
+            "config",
+            "plaintext",
+            "markdown",
+            "batch"

```

Comment:
I'm surprised to see this file change, and worried that someone else's changes will reformat again. I can accept this for now, but let's take care to watch out for frequent format changes - if we keep going back and forth with these little updates then we need to standardize something, somewhere, so that we stop getting changes.

Reply:
Agreed